jetty-server/src/main/java/org/eclipse/jetty/server/Request.java

```diff
@@ -241,6 +241,11 @@ public Request(HttpChannel channel, HttpInput input)
     }
 
     public HttpFields getHttpFields()
+    {
+        return getHeaders();
+    }
+
+    public HttpFields getHeaders()
```

Comment:
Why keep both methods?   If `getHeaders` is a better name, let's just use it.
If we really REALLY need both, then we need comments/javadoc saying why.

Reply:
@gregw this PR is for 10+ so in 10/11 we need to keep both methods, but in 12 we only have `getHeaders()` already.